index.html

```diff
@@ -0,0 +1,747 @@
+        <section class=random-section>
+            <ul class="three-column">
+                <li>
+                    <figure>
+                        <img src="assets/images/stone1.jpg" alt="stone">
+                        <figcaption>
+                            <span>Seeking Solar System’s Secrets, NASA’s OSIRIS-REX Mission Touches Bennu Asteroid</span>
+                            <span>1h ago</span>
+                        </figcaption>
+                    </figure>
+                </li>
+                <li>
+                    <figure>
+                        <img src="assets/images/stellar.jpg" alt="stone">
+                        <figcaption>
+                            A Black Hole’s Lunch: Stellar Spaghetti
+                            Oct. 17
+                        </figcaption>
+                    </figure>
+                </li>
+                <li>
+                    <figure>
+                        <img src="assets/images/rocket.jpg" alt="stone">
+                        <figcaption>
+                            Jeff Bezos’ Company Is Carrying Scientific Cargo to Space. It’s Not Amazon.
+                            Oct. 19
+                        </figcaption>
+                    </figure>
+                </li>
+            </ul>
+
+            <ul class="three-column">
+                <li>
+                    <figure>
+                        <img src="assets/images/spaceship.jpg" alt="stone">
+                        <figcaption>
+                            Jeff Bezos’ Company Is Carrying Scientific Cargo to Space. It’s Not Amazon.
+                            Oct. 19
+                        </figcaption>
+                    </figure>
+                </li>
+                <li>
+                    <figure>
+                        <img src="assets/images/nobel.jpg" alt="stone">
+                        <figcaption>
+                            Jeff Bezos’ Company Is Carrying Scientific Cargo to Space. It’s Not Amazon.
+                            Oct. 19
+                        </figcaption>
+                    </figure>
+                </li>
+                <li>
+                    <figure>
+                        <img src="assets/images/satelite.jpg" alt="stone">
+                        <figcaption>
+                            Jeff Bezos’ Company Is Carrying Scientific Cargo to Space. It’s Not Amazon.
+                            Oct. 19
+                        </figcaption>
+                    </figure>
+                </li>
+            </ul>
+        </section>
+        <div>
+            <p style="margin-left:20rem; font-weight: bolder; border-top: 2px solid black;width: 600px; padding-top: 1em;"> Editiors' Pick</p> 
+        </div>
+        <section class="random-section2">
+            <ul class="three-columns" style="display: flex;">
```

Comment:
Please, remove inline styling, it is considered a bad practice, move it to your CSS files.